### ISSUE TITLE
move.py - add email notification when transfer is complete

### ISSUE THREAD

**Issue by ecodonohoe** [kieranjol/IFIscripts#51]

**Comment by kieranjol** [comment]
Yup, awesome idea. Btw I made a different branch with some new change. I'll merge with the master branch once I'm sure it works. https://github.com/kieranjol/IFIscripts/tree/kieranjol-file-count
